sporkify/frontend/templates/inventory.html

```diff
@@ -76,6 +77,47 @@
 
     </form>
 
+    <div id="MiniHistoryGross">
+        <a href="/reports" target="_blank" style="color:black"><h1>Sales Reports</h1></a>
```

Comment:
Did you mean for this sales chart to be on the inventory (unsold) page?

Reply:
... i dunno.. TBD? i have to make 4 more charts sooo it'll find a home 🥉